modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -836,49 +828,46 @@ GrainTracker::updateFieldInfo()
   std::map<unsigned int, Real> tmp_map;
   MeshBase & mesh = _mesh.getMesh();
 
-  for (std::map<unsigned int, MooseSharedPointer<FeatureData> >::iterator grain_it = _unique_grains.begin(); grain_it != _unique_grains.end(); ++grain_it)
+  for (const auto & grain_kv : _unique_grains)
   {
-    unsigned int curr_var = grain_it->second->_var_idx;
+    unsigned int curr_var = grain_kv.second->_var_idx;
```

Comment:
There are several that I've missed, mostly on purpose... Give and old C++03 programmer some time! 😆 

There is another interesting thing going on in this line, and it's more of me just being a little uncomfortable with `auto`. Note the new name of the variable. I've gone from `grain_it` to `grain_kv` (grain key/value), instead of the simpler choice of just `grain`. I did that pretty consistently in all of the map iterators. I'm not sure I love it, maybe I just need to get over it. What do you guys think?